flink-runtime/src/main/java/org/apache/flink/runtime/rpc/RpcGateway.java

```diff
@@ -28,5 +28,6 @@
 	 *
 	 * @return Fully qualified address under which the associated rpc endpoint is reachable
 	 */
+	@NativeMethod
 	String getAddress();
```

Comment:
Can't we simply say that all methods defined in `RpcGateway` are implicitly a *native* method. Then we could save the `@NativeMethod` annotation.